apps/client/src/features/event-editor/EventEditor.tsx

```diff
@@ -16,23 +16,24 @@ export type EventEditorSubmitActions = keyof OntimeEvent;
 export type EditorUpdateFields = 'cue' | 'title' | 'presenter' | 'subtitle' | 'note' | 'colour';
 
 export default function EventEditor() {
-  const openId = useAppMode((state) => state.editId);
+  const { eventsToEdit } = useEventSelection();
   const { data } = useRundown();
   const { updateEvent } = useEventAction();
 
   const [event, setEvent] = useState<OntimeEvent | null>(null);
 
   useEffect(() => {
-    if (!data || !openId) {
+    if (!data) {
       setEvent(null);
       return;
     }
 
-    const event = data.find((event) => event.id === openId);
+    const event = data.find((event) => eventsToEdit.has(event.id));
```

Comment:
Would you mind explaining what the decision is here?

We have a set of events from the multi select `eventsToEdit`
The editor should open the event that is currently marked for edit

In the case where there are multiple events you have chosen to close the drawer.

It seems to me that, as of now, it will end up with the data from one of the events (the first?) and that we just dont see the problem because the `EventEditorExport` hides the drawer for us.
I believe this component will also be re-rendering offscreen as the multi-select changes

I believe the logic here should reflect the decision to only work with one event, this would make the logic simpler and cleanup the behaviour as well

Reply:
I was just porting old logic to my changes. Clearly, that's not gonna work. Thanks for pointing it out. You're suggesting we need to hide the drawer completely?

Reply:
I am not sure what the best way is.
The technical requirement is that, if the drawer is hidden, changes to the selection should be no-ops